src/Microsoft.Framework.Runtime.Roslyn/RoslynDiagnostics.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Framework.Runtime.Roslyn
+{
+    internal class RoslynDiagnostics
+    {
+        internal static readonly DiagnosticDescriptor StrongNamingNotSupported = new DiagnosticDescriptor(
```

Comment:
I'll check in to that... last I checked (this morning) we had to implement `StrongNameProvider` and something wasn't working, but maybe I missed something.